documentation/en/user/source/troubleshooting/index.rst

```diff
@@ -29,7 +29,19 @@ Note that you can safely modify gridsubsets, but the gridset should always be th
 Logging
 -------
 
-Logging can be controlled through :file:`WEB-INF/classes/log4j.properties`. By default, log messages will end up in the standard log of the container, which is for Apache Tomcat is inside the :file:`logs` directory.
```

Comment:
On this, I'll not that the code still has ``log4j.properties`` files lying around, including one in the final war file:

```
find . -name "log4j.properties"
./geowebcache/web/src/main/resources/log4j.properties
./geowebcache/web/target/geowebcache/WEB-INF/classes/log4j.properties
./geowebcache/web/target/classes/log4j.properties
./geowebcache/core/src/test/resources/log4j.properties
./geowebcache/core/target/test-classes/log4j.properties
./geowebcache/sqlite/src/test/resources/log4j.properties
./geowebcache/sqlite/target/test-classes/log4j.properties
```

Reply:
I see just now they all are in "target"... never mind!

Reply:
I will replace

Reply:
The individual modules have been configured to use `src/test/resources/logging.properties` during testing, only `web` uses a `log4j2.xml` for testing.

It was just much easier than trying to figure out how to get `GeoTools.init()` to be called by the various test cases.

Reply:
Marking this resolved as the docs have been updated with working examples.